page_example/page_example.module

```diff
@@ -72,15 +72,29 @@ function page_example_permission() {
 function page_example_menu() {
 
   // This is the minimum information you can provide for a menu item. This menu
-  // item will be created in the default menu, usually Navigation.
-  $items['examples/page_example'] = array(
-    'title' => 'Page Example',
+  // item will be created in the default internal menu, which is invisible in the 
+  // user interface.
+  $items['examples/page_menu_internal'] = array(
```

Comment:
It is not necessary to change the route path. It can still be `'examples/page_example'`. This, and other similar routes used by sub-modules should be changed to make the sub-modules accessible from the user interface, which is the task for another issue.